### PR TITLE
feat: twitch clip date and feature

### PR DESCRIPTION
**Pull request by CKY-**
### Description of the Change

this adds 2 check boxes to the play random clip from twitch 
one for featured only 

one for max age with max age the time input was added so that users can select the amount of time to the current time to get clips from the api request.

and fixed linting
### Applicable Issues

#2594
#1554

### Testing

while coding and playing with this i was randomly targeting chat users 

### Screenshots

image

### CODE REVIEW

File: src/backend/effects/builtin/play-video.js [hunk 1]
```diff
@@ -124,6 +124,38 @@ const playVideo = {
...
+                class="mt-6 mb-2"
+                ng-class="{'has-error': $ctrl.formFieldHasError('clipSeconds')}"
+            >
+                <div class="form-group flex-row jspacebetween" style="margin-bottom: 0;">
+                    <div>
+                        <label class="control-fb control--checkbox" style="margin:0;"> Maximum Clip Age
+                        <input type="checkbox" ng-model="effect.useMaxClipAge">
+                            <div class="control__indicator"></div>
+                        </label>
+                    </div>
```
Comment (ebiggz) [inline src/backend/effects/builtin/play-video.js:137]: Please use the `<firebot-checkbox />` component for this bad boy. I know theres probably other checkboxes in this effect not using the "old way", I'm just trying to make sure at least that any new checkboxes we add are using the component :)

File: src/backend/effects/builtin/play-video.js [hunk 2]
```diff
@@ -124,6 +124,38 @@ const playVideo = {
...
+                </div>
+                    <div class="mt-6 form-group flex-row jspacebetween" style="margin-bottom: 0;">
+                        <div>
+                            <label class="control-fb control--checkbox" style="margin:0;"> Only Featured Clips
+                                <input type="checkbox" ng-model="effect.isFeatured">
+                                <div class="control__indicator"></div>
+                            </label>
+                        </div>
+                    </div>
+            </div>
```
Comment (ebiggz) [inline src/backend/effects/builtin/play-video.js:158]: Can use `<firebot-checkbox />` here as well

**Review by ebiggz — CHANGES_REQUESTED** [review]

**Review by SReject — APPROVED** [review]

**Review by ebiggz — APPROVED** [review]

**Comment by angsaysroar** [comment]
@CKY- this is look really spiffy!

**Comment by angsaysroar** [comment]
I noticed it doesn’t account for when someone does not have featured clips, so maybe add else if clause if the effect has thee maximum age of a clip play that else play a random clip. 

I hope that made sense. Lol

**Pull request merged**